.circleci/config.yml

```diff
@@ -443,114 +481,49 @@ jobs:
-          command: |
-            echo -e "The logs will be saved to \"/tmp/spark_fuzzer.log\" and uploaded" \
-                    "as a job artifact (check the \"Artifacts\" tab above)."
-            _build/debug/velox/expression/tests/spark_expression_fuzzer_test \
-                --seed ${RANDOM} \
-                --duration_sec 600 \
-                --logtostderr=1 \
-                --minloglevel=0 \
-              2>&1 | tee /tmp/spark_fuzzer.log || ( \
-                tail -n 1000 /tmp/spark_fuzzer.log;
-                echo "FAIL: Fuzzer run failed";
-                exit 1; \
-              )
-            echo -e "\n\nSpark Fuzzer run finished successfully."
-          no_output_timeout: 20m
-      - store_artifacts:
-          path: '/tmp/spark_fuzzer.log'
-      - post-steps
+      - fuzzer-runs:
+          fuzzer_output: "/tmp/spark_fuzzer.log"
+          fuzzer_name: "Spark"
+          fuzzer_exe: "_build/debug/velox/expression/tests/spark_expression_fuzzer_test"
+          fuzzer_args: " --seed ${RANDOM} --duration_sec 600 --logtostderr=1 --minloglevel=0 "
+
 
   linux-aggregate-fuzzer-run:
     executor: build
     steps:
-      - pre-steps
-      - run:
-          name: Build
-          command: |
-            make debug NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
-            ccache -s
-          no_output_timeout: 2h
-      - run:
-          # Run Aggregate Fuzzer, save the output to /tmp/aggregate_fuzzer.log, and upload the file
-          # as job artifact to make it convenient for users to inspect it -
-          # these logs can also be quite large. Always prints the last 1k lines to
-          # the job output for context.
-          name: "Run Aggregate Fuzzer Tests"
-          command: |
-            echo -e "The logs will be saved to \"/tmp/aggregate_fuzzer.log\" and uploaded" \
-                    "as a job artifact (check the \"Artifacts\" tab above)."
-            _build/debug/velox/exec/tests/velox_aggregation_fuzzer_test \
-                --seed ${RANDOM} \
-                --duration_sec 3600 \
-                --logtostderr=1 \
-                --minloglevel=0 \
-              2>&1 | tee /tmp/aggregate_fuzzer.log || ( \
-                tail -n 1000 /tmp/aggregate_fuzzer.log;
-                echo "FAIL: Aggregate Fuzzer run failed";
-                exit 1; \
-              )
-            echo -e "\n\nAggregate Fuzzer run finished successfully."
-          no_output_timeout: 120m
-      - store_artifacts:
-          path: '/tmp/aggregate_fuzzer.log'
-      - post-steps
+      - fuzzer-runs:
+          fuzzer_output: "/tmp/aggregate_fuzzer.log"
+          fuzzer_name: "Aggregate"
+          fuzzer_exe: "_build/debug/velox/exec/tests/velox_aggregation_fuzzer_test"
+          fuzzer_args: " --seed ${RANDOM} --duration_sec 3600 --logtostderr=1 --minloglevel=0 "
+
+  linux-join-fuzzer-run:
+    executor: build
+    environment:
+      VELOX_DEPENDENCY_SOURCE: SYSTEM
```

Comment:
Why do we need this for the join fuzzer, but not for aggregate fuzzer?

Reply:
And probably should be added xor not added consistently

Reply:
@assignUser Thank you for explaining. Should we decide whether we want to use this or not and make sure to apply the decision to all fuzzer jobs consistently?

Reply:
I think it would make sense to set it to `SYSTEM` for all fuzzer jobs as we don't want to spend any time on building deps from source and if something is missing from the image we should have the build error instead of wasting time build the dependency!